todo_app/data/trello_items.py

```diff
@@ -0,0 +1,181 @@
+# Trello card actions
+#
+import requests
+import json
+from todo_app.data.Item_class import Item
+
+_DEFAULT_ITEMS = [
+    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
+    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
+]
+
+lists_ids=[]
+items=[]
```

Comment:
I don't think you ever actually use this variable since you override it in each method anyway